documentation/tarron_notes.md

```diff
@@ -1,5 +1,15 @@
 # Tarron's Notes
 
+## Questions
+
+- [ ] What does this notation `(rootQuery, { planId }, context)` mean in lambda function declaration
```

Comment:
That's regular function notation. The first parameter is the rootQuery object. The second is the query's arguments. That is destructured into the variable planId. Context is passed as the third parameter to every query and mutation.

Reply:
Huh. I didn't know you could destructure within a function signature like that.